substrate/frame/staking/src/pallet/impls.rs

```diff
@@ -1384,14 +1384,14 @@ impl<T: Config> historical::SessionManager<T::AccountId, Exposure<T::AccountId,
 	) -> Option<Vec<(T::AccountId, Exposure<T::AccountId, BalanceOf<T>>)>> {
 		<Self as pallet_session::SessionManager<_>>::new_session_genesis(new_index).map(
 			|validators| {
-				let current_era = Self::current_era()
+				let current_era = CurrentEra::<T>::get()
 					// Must be some as a new era has been created.
 					.unwrap_or(0);
 
 				validators
 					.into_iter()
 					.map(|v| {
-						let exposure = Self::eras_stakers(current_era, &v);
+						let exposure = ErasStakersClipped::<T>::get(current_era, &v);
```

Comment:
Why is it changed here? It is not a getter.
```suggestion
						let exposure = Self::eras_stakers(current_era, &v);
```

Reply:
I must have thought that eras_stakers resolves to ErasStakersClipped::<T>::get but not looking at it again, it doesn't seem like it. Good catch.